charts/eks-operator/templates/deployment.yaml

```
@@ -25,3 +25,15 @@ spec:
           value: {{ .Values.httpsProxy }}
         - name: NO_PROXY
           value: {{ .Values.noProxy }}
+{{- if .Values.additionalTrustedCAs }}
+        volumeMounts:
+          - mountPath: /etc/ssl/certs/ca-additional.pem
+            name: tls-ca-additional-volume
+            subPath: ca-additional.pem
```

The current code will result in creating a file at this full path: `/etc/ssl/certs/ca-additional.pem/ca-additional.pem`

```suggestion
          - mountPath: /etc/ssl/certs
            name: tls-ca-additional-volume
```

---

I think the desired result is to create the file.

---

this is not an issue, see
https://github.com/rancher/aks-operator/pull/18/files#r641063779